xml/hwbp_registers.xml

```diff
@@ -149,8 +149,8 @@
     <register name="Trigger Info" short="tinfo" address="0x7a4">
         This register is optional if no triggers are implemented, or if
         \FcsrTdataOneType is not writable and \FcsrTinfoVersion would be 0. In
-        this case the debugger can read the only supported type from
-        \RcsrTdataOne.
+        this case reading this register must return 0 or raise an exception, and
```

Comment:
```suggestion
        this case reading this register must return 0 or raise an illegal instruction exception, and
```

Reply:
Also, I think that it's ambiguous whether it must (return 0 OR (exception AND debugger can ...)) vs. ((return 0 OR exception) AND debugger can ...).  The comma might be intended to help but it's not obvious to me until I think about it some more.  Maybe two sentences: It must return 0 or exception.  If either of those happen then the debugger can ...

Reply:
Agreed about 2 sentences.

How strongly do you feel about "illegal instruction exception?" I was deliberately vague because the hypervisor extension mandates that CSRs that appear not to exist to a guest must raise a virtual instruction exception. But maybe we should just ignore that possibility, because I bet there are other places in the spec where we don't think about that scenario.

Reply:
Sure, we can leave the type of exception off if you want.  That requires the reader to figure it out but it's not rocket science.  It's not like any sane reader would expect a page fault.